requirements.txt

```
@@ -1,44 +1,4 @@
-appdirs==1.4.4
-autograd==1.6.2
-autoray==0.6.8
-cachetools==5.3.3
-certifi==2024.2.2
-cffi==1.16.0
-charset-normalizer==3.3.2
-cryptography==42.0.5
-Cython==3.0.8
-dill==0.3.8
-future==1.0.0
-idna==3.6
-mpmath==1.3.0
-networkx==3.2.1
-ninja==1.11.1.1
-ntlm-auth==1.5.0
-numpy==1.26.4
-orjson==3.9.15
-pbr==6.0.0
-pennylane==0.34
-PennyLane-Lightning==0.34
-ply==3.11
-psutil==5.9.8
-pycparser==2.21
-python-constraint==1.4.0
-python-dateutil==2.8.2
-qiskit==0.45.3
-qiskit-aer==0.13.3
-qiskit-ibm-runtime==0.20.0
-qiskit-ibm-provider==0.10.0
-qiskit-ignis==0.7.1
-qiskit-terra==0.45.3
-requests==2.31.0
-requests-ntlm==1.2.0
-retworkx==0.14.1
-scipy==1.12.0
-semantic-version==2.10.0
-six==1.16.0
-stevedore==5.2.0
-symengine==0.11.0
-sympy==1.12
-toml==0.10.2
-urllib3==2.2.1
-websocket-client==1.7.0
+pennylane>=0.32
+qiskit
+numpy
+sympy
```

```suggestion
sympy

```
Do we need to be pinning any of this?

---

I don't think so... the file used for the ci-tests are not pinned either so I assume that it shouldn't be needed.